ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/disktype/disksize/provision/cucushift-installer-rehearse-azure-ipi-disktype-disksize-provision-chain.yaml

```diff
@@ -8,6 +8,6 @@ chain:
   - ref: enable-qe-catalogsource
   - chain: cucushift-installer-check
   - chain: cucushift-installer-check-azure
-  - ref: cucushift-installer-check-azure-disksize
+  - ref: cucushift-installer-check-azure-disk-size
```

Comment:
I am thinking if we even can merge this step to `cucushift-installer-check-azure-disk` ?